dll/ntdll/ldr/ldrpe.c

```diff
@@ -1028,14 +1028,14 @@ LdrpSnapThunk(IN PVOID ExportBase,
         /* Is this a static snap? */
         if (Static)
         {
+            RtlInitAnsiString(&TempString, DllName ? DllName : "Unknown");
             /* Inform the debug log */
             if (IsOrdinal)
-                DPRINT1("Failed to snap ordinal 0x%x\n", OriginalOrdinal);
+                DPRINT1("Failed to snap ordinal %Z!0x%x\n", &TempString, OriginalOrdinal);
```

Comment:
Why not simply %s and DllName?
I don't see the advantage of the ANSI_STRING here.

Reply:
The ANSI_STRING is already used below, and it already did the NULL check for us.